soccer/Processor.cpp

```diff
@@ -382,7 +382,8 @@ void Processor::run() {
         // Read radio reverse packets
         _radio->receive();
 
-        for (Packet::RadioRx& rx : _radio->reversePackets()) {
+        while (_radio->hasReversePackets()) {
```

Comment:
(I haven't read too much into this code, but), I think this was iterating in order (low->high index) before, and now this is iterating in reverse (high->low index). I'm worried this will change the value of `curStatus.lastRadioRxTime`.

If we decide that makes a difference, maybe we can use a queue instead of a vector for storing these packets.